.github/workflows/ci.yml

```diff
@@ -104,6 +104,18 @@ jobs:
       with:
         src: "../test/${{ matrix.copier_config.foldername }}/src"
    
+    - name: mypy checks basic
+      if: ${{ contains(matrix.copier_config.extra_flags, 'mypy_type_checking=basic') && !contains(matrix.copier_config.extra_flags, 'create_example_module=no') }}
+      run: |
+        cd "../test/${{ matrix.copier_config.foldername }}"
+        mypy src tests
+   
+    - name: mypy checks strict
```

Comment:
I don't think we have a configuration for strict mypy case. Do you think it's worth changing one of the existing configurations so that we are exercising this?

Reply:
We do not! Either we could add one (your bloat concern relevant) or modify one or assume it isn't terribly important.
One thing I also discovered during this exercise was that:
  mypy tests            fails as I described to you
  mypy src tests     passes presumably because it knows about src before processing tests. Feature or bug?